rnd/autogpt_server/autogpt_server/data/block.py

```diff
@@ -8,7 +8,9 @@
 
 from autogpt_server.util import json
 
-BlockData = dict[str, Any]
+BlockInput = dict[str, Any]
+BlockData = tuple[str, Any]
+BlockOutput = Generator[BlockData, None, None]
```

Comment:
Adding type definitions adds another layer of complexity for understanding what types are being used. 

I'd prefer just using the types directly.

Reply:
This type can be replaced to be an actual class anytime soon, so keeping it named is really required for this.

<img width="689" alt="image" src="https://github.com/Significant-Gravitas/AutoGPT/assets/76959103/247fc66a-c0c7-4c79-bf45-82d4233d242a">

If you hover on types, you can easily see the type.